specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -7368,6 +7686,32 @@
         }
       },
       "x-ms-discriminator-value": "A2A"
+    },
+	"A2ARemoveDisksInput": {
+      "description": "A2A remove disk(s) input.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RemoveDisksProviderSpecificInput"
+        }
+      ],
+      "properties": {
+        "vmDisksUris": {
```

Comment:
is naming fine DisksUris?

Reply:
property names should also be camel cased.

Reply:
vmDisksUris is camel cased, did not get the comment.
Suggesting a change?

Reply:
yes, this is correct. I thought you asked a question and I replied saying es, propertynames should also be camel cased.